packages/app/components/Settings/ChatbotParameter.tsx

```diff
@@ -53,13 +53,7 @@ const ChatbotParameter: React.FC<ChatbotParameterProps> = ({
     {},
   )
 
-  useEffect(() => {
-    if (visible) {
-      fetchChatbotSettings()
-    }
-  }, [visible])
-
-  const fetchChatbotSettings = async () => {
+  const fetchChatbotSettings = useCallback(async () => {
```

Comment:
I didn't really want to touch the chatbot stuff, but it wouldn't let me complete my merge unless i fixed these eslint errors (hooks missing dependencies, unused imports)